.buildkite/browser-pipeline.yml

```diff
@@ -231,6 +336,68 @@ steps:
     concurrency: 5
     concurrency_group: 'browserstack'
 
+  - label: ':iphone: iOS 15.4 Browser tests'
+    depends_on: "browser-maze-runner-image"
+    timeout_in_minutes: 20
+    plugins:
+      docker-compose#v3.9.0:
+        pull: browser-maze-runner
+        run: browser-maze-runner
+        use-aliases: true
+        verbose: true
+        command:
+          - --farm=bs
+          - --browser=iphone_13
+    env:
+      HOST: "bs-local.com"
+    concurrency: 5
+    concurrency_group: 'browserstack'
+
+  - label: ':android: Google Nexus 5 Browser tests'
```

Comment:
Could we called these "Android X browser tests" rather than after the device model.  It's the Android version that we care about.